omniauth-auth0.gemspec

```diff
@@ -21,8 +21,10 @@ omniauth-auth0 is the OmniAuth strategy for Auth0.
   s.executables   = `git ls-files -- bin/*`.split('\n').map{ |f| File.basename(f) }
   s.require_paths = ['lib']
 
+  s.add_runtime_dependency 'omniauth', '~> 1.9'
```

Comment:
Good catch @frederikprijck. Yes, looks like those 3 PRs are features & fixes that were never released, and as such, should be on a _minor_ release rather than a _patch_. Also, the change in the `omniauth` [dependency version](https://github.com/auth0/omniauth-auth0/pull/121/files#diff-e5ba7bdf9266df5c25bb52408e772964c007af0dfb9f3b5670aac3bb2bee9aafR24) should go in a separate PR rather than in this same "release PR". 

@davidpatrick please create a separate PR for this dependency change. (Or reuse this one and create the release in a separate PR)

Reply:
nope its part of this release